recipe/meta.yaml

```diff
@@ -7,17 +7,18 @@ package:
 source:
   fn: cmt-v{{ version }}.tar.gz
   url: https://github.com/lucastheis/cmt/archive/v{{ version }}.tar.gz
-  sha1: 7482caefa5359a351ac834920a092b5594098d70
+  sha256: 252d1029c62889a533d50e017445407e1de653cac8bee48ecdc34e4d8fd6e7ce
   patches:
-    # Workaround to ensure `gcc` compiler is used on Mac.
-    # See issue ( https://github.com/lucastheis/cmt/issues/10 ).
-    # No patch submitted as this will require discussion with the developer.
+    # Patch to avoid passing `-stdlib` to the compiler generally
+    # See issue ( https://github.com/lucastheis/cmt/issues/25 ).
+    # Patch for this has been submitted.
+    # See pull request ( https://github.com/lucastheis/cmt/pull/27 ).
     - setup.py.patch
-    # Incorrect import here makes tests fail and reduces functionality.
-    # See issue ( https://github.com/lucastheis/cmt/issues/14 ).
+    # Patch to register missing Python method added in 0.6.0.
+    # See issue ( https://github.com/lucastheis/cmt/issues/29 ).
     # Patch for this has been submitted.
-    # See pull request ( https://github.com/lucastheis/cmt/pull/18 ).
-    - images.py.patch
```

Comment:
As noted already, was fixed by commit ( https://github.com/lucastheis/cmt/commit/a5d2a33b485068468bd8a8c92a55488ac79fba21 ). So this patch is no longer necessary.